man/os-release.xml

```diff
@@ -325,6 +325,29 @@
 
           <xi:include href="version-info.xml" xpointer="v249"/></listitem>
         </varlistentry>
+
+        <varlistentry>
+          <term><varname>RELEASE_TYPE=</varname></term>
+
+          <listitem><para>A lower-case string (no spaces or other characters outside of 0-9, a-z, ".",
+          "_", and "-"), describing what kind of release this version of the OS is. Known values follow:
+          <literal>stable</literal> is for normal releases of the system, suitable for production use.
+          Generally, stable releases become end-of-life soon after the next major stable release is out.
+          Examples include Fedora 40, Debian 12 Bookworm, Ubuntu 23.10, OpenSUSE Tumbleweed, and Arch Linux.
```

Comment:
Debian releases are actually LTS, as they fit the description below - ie, Debian 10 and 11 are still supported and receive security updates